src/bin/cql-stress-cassandra-stress/settings/command/help.rs

```diff
@@ -0,0 +1,75 @@
+use anyhow::Result;
+
+use super::{read_write::print_help_read_write, CommandParams, ParsePayload};
+
+pub fn print_help() {
+    println!(
+"Usage:      cassandra-stress <command> [options]
+Help usage: cassandra-stress help <command|option>
+
+---Commands---
+read                 : Multiple concurrent reads - the cluster must first be populated by a write test
+write                : Multiple concurrent writes against the cluster
+mixed                : Interleaving of any basic commands, with configurable ratio and distribution - the cluster must first be populated by a write test
+counter_write        : Multiple concurrent updates of counters.
+counter_read         : Multiple concurrent reads of counters. The cluster must first be populated by a counterwrite test.
+user                 : Interleaving of user provided queries, with configurable ratio and distribution
+help                 : Print help for a command or option
+print                : Inspect the output of a distribution definition
+legacy               : Legacy support mode
+version              : Print the version of cassandra stress
+
+---Options---
+-pop                  : Population distribution and intra-partition visit order
+-insert               : Insert specific options relating to various methods for batching and splitting partition updates
+-col                  : Column details such as size and count distribution, data generator, names, comparator and if super columns should be used
+-rate                 : Thread count, rate limit or automatic mode (default is auto)
+-mode                 : Thrift or CQL with options
+-errors               : How to handle errors when encountered during stress
+-schema               : Replication settings, compression, compaction, etc.
+-node                 : Nodes to connect to
+-log                  : Where to log progress to, and the interval at which to do it
+-transport            : Custom transport factories
+-port                 : The port to connect to cassandra nodes on
+-sendto               : Specify a stress server to send this command to
+-graph                : Graph recorded metrics
+-tokenrange           : Token range settings
+-cloudconf            : Configuration file of a serverless Scylla cluster to connect to"
```

Comment:
See the next comment.

Reply:
Fixed - still would like to receive a feedback about new solution, tho.

Introduced:
- static array `SUPPORTED_OPTIONS` in `settings::command::mod.rs', which holds currently supported commands. Not sure if this is the right solution. The best solution would be to iterate over `Command` enum's values, but it would require importing new crate: which provides such mechanism https://crates.io/crates/strum. Not sure it that's worth it, tho.
- `Command::print_supported_commands`
- `Command::print_short_description`
- `Command::print_help`